crates/sui-types/src/storage.rs

```diff
@@ -53,6 +53,12 @@ pub enum DeleteKind {
     Wrap,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Serialize, Deserialize)]
+pub enum MarkerKind {
```

Comment:
Please document each kind with their precise semantics (i.e. what does it mean when there is such a marker with a certain key)